src/components/ProfileFormUpdate/index.js

```diff
@@ -0,0 +1,170 @@
+        <Mutation
+          mutation={EDIT_PROFILE_MUTATION}
+          // update={(store, { data: { updateDailyReport } }) => {
+          //   if (store.data.data.ROOT_QUERY.userReports) {
+          //     const data = store.readQuery({ query: DAILY_REPORTS_QUERY });
+          //     data.userReports.dailyReports = data.userReports.dailyReports.map(
+          //       report => (report.id === updateDailyReport.id ? updateDailyReport : report)
+          //     );
+          //     store.writeQuery({ query: DAILY_REPORTS_QUERY, data });
+          //   }
+          // }}
+        >
+          {(editProfile, { loading, error }) => (
+            <CenterWrapper>
+              {error && <div>error</div>}
+
+              <Formik
+                initialValues={{
+                  address,
+                  phone,
+                  email,
+                  avatar
+                }}
+                enableReinitialize
+                onSubmit={async (values, { setSubmitting, setStatus, setErrors }) => {
+                  const { address, phone, email, avatar } = values;
+
+                  try {
+                    await editProfile({
+                      variables: {
+                        id,
+                        address,
+                        phone,
+                        email,
+                        avatar
+                      }
+                    });
+                    setStatus({ success: true });
+                  } catch (error) {
+                    setStatus({ success: false });
+                    setSubmitting(false);
+                    setErrors({ submit: error.message });
+                  }
+                }}
+                render={({ values, handleSubmit, handleChange, touched, errors, status }) => (
+                  <Form
+                    onSubmit={handleSubmit}
+                    loading={loading}
+                    success={status ? status.success : false}
+                  >
+                    <TextInput
+                      type="text"
+                      label="Address"
+                      name="address"
+                      defaultValue={values.address}
+                      error={touched.address && errors.address}
+                      onChange={handleChange}
+                    />
+
+                    <TextInput
+                      type="text"
+                      label="Phone"
+                      name="phone"
+                      defaultValue={values.phone}
+                      error={touched.phone && errors.phone}
+                      onChange={handleChange}
+                    />
+
+                    <TextInput
+                      type="text"
+                      label="Email"
+                      name="email"
+                      defaultValue={values.email}
+                      error={touched.email && errors.email}
+                      onChange={handleChange}
+                    />
+
+                    <TextInput
```

Comment:
Do we update `avatar`? That means we will need to use some file hosting service.
Like AWS S3 or Cloudinary or something.

Reply:
Currently, I just allow update avatar by passing URL. We will improve in the future.

Reply:
Ah OK, I got it!